terraform/validation/main.tf

```diff
@@ -49,6 +49,7 @@ data "template_file" "docker_compose" {
     incoming_packets_alarm = var.incoming_packets_alarm
 
     cortex_instance_endpoint = var.cortex_instance_endpoint
+    rollup                   = var.rollup
```

Comment:
What does roll-up for? Any documents on that since I could not fine one.

Reply:
@khanhntd  Hi, rollup is a new feature for cwmetricsvalidator to enable or disable dimension rollup for expected test templates. By default it was set to true earlier, but with this newly implemented feature we can also turn it off.

Reply:
Reasonable. Thanks for explaining.